### ISSUE TITLE
Not all Querry annotated

### ISSUE THREAD

**Issue by nancytoy** [eggnogdb/eggnog-mapper#334]
Hello every one, 

I need help please!!!

I made functional annotation for a set of 30,000 transcriptomes with eggnog mapper (emapper-2.1.3, using command lines)
 and in the result only 500 querry were annotated

why don't I have all the querry annotated ?? how can I fix it ??

Thank you
Nancy

**Comment by Cantalapiedra** [comment]
Hi @nancytoy ,

What is the command line you are running?
Do you mean 30,000 transcriptomes or 30,000 transcripts?

Thank you.

Best,
Carlos

**Comment by nancytoy** [comment]
Hi,
Thank you for your answer !

30,000 Transcriptomes, I used this commandline:
emapper.py -i FASTA_FILE_PROTEINS -o File

Best
Nancy

**Comment by Cantalapiedra** [comment]
Hi @nancytoy ,

are your input data proteins? Or CDS? or transcripts?

If your input data are CDS then you may need to use `--itype CDS --translate`.
Check https://github.com/eggnogdb/eggnog-mapper/wiki/eggNOG-mapper-v2.1.2-to-v2.1.4#input-data-options

The translation will be carried out with the default translation table unless you use `--trans_table TRANS_TABLE_CODE`.
Check https://github.com/eggnogdb/eggnog-mapper/wiki/eggNOG-mapper-v2.1.2-to-v2.1.4#gene-prediction-options

If your input data are transcripts you may need to use `--genepred`, gene prediction options, along with `--itype genome` or `--itype metagenome`. Check https://github.com/eggnogdb/eggnog-mapper/wiki/eggNOG-mapper-v2.1.2-to-v2.1.4#gene-prediction-options

You could also run gene prediction with other software and use the predicted CDS/proteins as input for eggnog-mapper.

I hope this makes sense.

Best,
Carlos

**Comment by nancytoy** [comment]
Hello, sorry I'am a begginer with bioinformatique :(
I'am using predected protein obtained from CDS

**Comment by Cantalapiedra** [comment]
Hi,

don't worry. 
If you are using proteins as input then it is weird that you are getting just a few annotations.
Would you mind sharing one of your input files, or at least a subsample, either here or to cpcantalapiedra@gmail.com?

Thank you.

Best,
Carlos

**Comment by nancytoy** [comment]
Hello, 
Here are some sequences of the fasta file that I want to annotate :

>TRINITY_GG_4228_c11_g1_i1.p1
MRVTIIARFFLASFLPAAMTLEMREPPNHSGRQKMGQEVSPSVIKAIDKLAKGMDERMEQRASLIDELKGSLEALAVKRGQMVVVSDKVARGVLQTIAQLRTIRDDHERLVLEELMNLNFAVKKLVTLRSRHSMLGMKDAERIEGSIRHSLRSKTEGTDSAGTPTWDGTSKDLKKGLPSSASGVVPKKLVVQAIESFMGGANRKQSEALSTVNAITLNAHKLFGQARRTVIMDVLSAPKNSQQPPLSTASTASYSIAKAMCDMDVCLSNDSKTMLESLQELKSVFGSHLVKDKDLSSSWTESQLETFKKELKLVEPQTKQESLLPMTRAPNSPITSYRALFRTVLQQLGNEDLSTLDPLPPSYSGSTSSLLNEPNYDQLLANVEENNKETLELIRSITRDVEEYVESQSSLQRKAYVMNRDNENMPSEGPLSDENKEEESGDIKDESTQDSSSPHSEVSADGLFVVSEYQFEPRSPSIKQPVNDNERKEDQIEKEATADGLFVVEEVDFLASNRMEAARNNTSSSHYNVASSNEDDIDAGIRRAHDDTMGIISNISDRVKDLLEKNRARQQHRGWEWGRTDDAEWEETELTQAILGSIFTDENK*
>TRINITY_GG_4228_c11_g1_i2.p1
MRVTIIARFFLASFLPAAMTLEMREPPNHSGRQKMGQEVSPSVIKAIDKLAKGMDERMEQRASLIDELKGSLEALAVKRGQMVVVSDKVARGVLQTIAQLRTIRDDHERLVLEELMNLNFAVKKLVTLRSRHSMLGMKDAERIEGSIRHSLRSKTEGTDSAGTPTWDGTSKDLKKGLPSSASGVVPKKLVVQAIESFMGGANRKQSEALSTVNAITLNAHKLFGQARRTVIMDVLSAPKNSQQPPLSTASTASYSIAKAMCDMDVCLSNDSKTMLESLQELKSVFGSHLVKDKDLSSSWTESQLETFKKELKLVEPQTKQESLLPMTRAPNSPITSYRALFRTVLQQLGNEDLSTLDPLPPSYSGSTSSLLNEPNYDQLLANVEENNKETLELIRSITRDVEEYVESQSSLQRKAYVMNRDNENMPSEGPLSDENKEEESGDIKDESTQDSSSPHSEVSADGLFVVSEYQFEPRSPSIKQPVNDNERKEDQIEKEATADGLFVVEEVDFLASNRMEAARNNTSSSHYNVASSNEDDIDAGIRRAHDDTMGIISNISDRVKDLLEKNRARQQHRGWEWGRTDDAEWEETELTQAILGSIFTDENK*
>TRINITY_GG_4228_c122_g1_i1.p1
KRTNVLMLLRKDPDDIQRMDLATLEDFSVINLSLTERRALYEHLRPCGIKWDKECSRFKVMHEGNWPLLEIPFEKRAWYNKMKGGLKENVALYERDTAQNGPADKEIDYLSGDYGWTEEA


Best 
Nancy

**Comment by Cantalapiedra** [comment]
Hi @nancytoy ,

sorry for the delay answering.

I tested those 3 sequences you shared and I had 1 out of 3 annotated (TRINITY_GG_4228_c122_g1_i1.p1) although the annotation fields were very scarce, with hit to Phatr44467, which has no function annotated in eggnog DB (eggnogdb.embl.de).

I don't know if this is what you would expect.

I tested the first sequence against uniprot and I had hits, but with very low identity and less than half coverage:

https://www.uniprot.org/blast/uniprot/B202109094ABAA9BC7178C81CEBC9459510EDDEA300DF65X?alignment=1

![image](https://user-images.githubusercontent.com/17877102/132711907-a8562daa-2fa3-48fe-819a-41fe6bdda473.png)

Therefore, maybe running emapper with more sensitivity may help (check https://github.com/eggnogdb/eggnog-mapper/wiki/eggNOG-mapper-v2.1.5#Diamond_search_options).

Also, the origin of your samples and the assembly could be involved.

Sorry to not be of more help.

Best,
Carlos
